src/com/hms/flexyosisoftconnector/Tag.java

```diff
@@ -120,6 +170,264 @@ public void recordTagValue(String time)
       }
    }
 
+   /**
+    * exportBlockDescriptor
+    * send an EBD command with a start and end time and a tag name.
+    * start time is stored in an external file, end time is current time.
+    */
+   public void exportBlockDescriptor()
+   {
+      // time pointers to keep track of start and end of queue
+      String startTime = "null";
+      String endTime = "null";
+
+      // get the time pointer to use in EBD call for the start of historical logs
+      startTime = readTimePointerFile();
+
+      // check that the error value did not get returned. Log if it did, and do not make EBD call.
+      if (startTime.equalsIgnoreCase("-1")) {
+         Logger.LOG_ERR("EBD Error: Start time not set after reading time pointer file.");
+         return;
+      }
+
+      // end time should be the current time to always keep it up to date
+      endTime = convertTimeIntHelper(System.currentTimeMillis());
+
+      String tagName = getTagName();
+
+      try {
+          // https://ewonsupport.biz/ebd/ <- has a nice EBD string builder. :)
+          Exporter exporter = new Exporter("$dtHL$ftT$st"+startTime+"$et"+endTime+"$fnvalues$tn"+tagName);
+          exporter.ExportTo("file:///"+tagDataFileName);

```

Comment:
whats up with the triple ///

Reply:
That was in the doc's example for using exportTo, so i just tried it that way and it works.. it's 4 slashes with one in the variable. I would guess they use forward slash to escape instead of backslash?

Reply:
So it does not work with just //?

Reply:
looks like it does work with just //
I should of tried that first

Reply:
Since Flexy supports file://, it shouldn't matter, but there's a difference between file:// and file:///. 

The syntax with file:// should be file://HOSTNAME/path and for file:/// should be file:///path.

file:/// is basically a short version of file://localhost/path, and is meant for localhost file access.